pyhiveapi/apyhiveapi/api/hive_api.py

```diff
@@ -40,7 +41,28 @@ def __init__(self, hiveSession=None, websession=None, token=None):
         self.session = hiveSession
         self.token = token
 
-    def request(self, type, url, jsc=None, camera=False):
+        self.homeID = None
+        if self.session is not None:
+            self.homeID = self.session.config.homeID
+
+    def getParams(self, products=False, devices=False, actions=False):
```

Comment:
I did some testing of the the sync code and the api didn't seem to work with the params when they where booleans. When I changed them to be lower case f and in quotes the api then worked as expected.
```suggestion
    def getParams(self, products='false', devices='false', actions='false\):
```

The other option would be to the leave the params as booleans and update the dictionary to be as per below converting them to string and lowercasing them.

```
params = {
            "products": str(products).lower(),
            "devices": str(devices).lower(),
            "actions": str(actions).lower(),
        }
```